apiVersion: events.k8s.io/v1
deprecatedCount: 1
deprecatedFirstTimestamp: "2023-03-08T11:51:57Z"
deprecatedLastTimestamp: "2023-03-08T11:51:57Z"
deprecatedSource:
  component: kubelet
  host: vrutkovs-mrmwt-master-0.c.openshift-gce-devel.internal
eventTime: null
kind: Event
metadata:
  creationTimestamp: "2023-03-08T11:51:57Z"
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:count: {}
      f:firstTimestamp: {}
      f:involvedObject: {}
      f:lastTimestamp: {}
      f:message: {}
      f:reason: {}
      f:source:
        f:component: {}
        f:host: {}
      f:type: {}
    manager: kubelet
    operation: Update
    time: "2023-03-08T11:51:57Z"
  name: apiserver-c9c8564b9-xv878.174a6f6b25e74845
  namespace: openshift-oauth-apiserver
  resourceVersion: "18747"
  uid: f3faefc4-e8da-4f85-ba42-0d9e0aeeebef
note: "Readiness probe error: Get \"https://10.129.0.29:8443/readyz\": dial tcp 10.129.0.29:8443:
  connect: connection refused\nbody: \n"
reason: ProbeError
regarding:
  apiVersion: v1
  fieldPath: spec.containers{oauth-apiserver}
  kind: Pod
  name: apiserver-c9c8564b9-xv878
  namespace: openshift-oauth-apiserver
  resourceVersion: "13632"
  uid: 3f956ae5-fcea-4f3c-973b-88255bc4aa78
type: Warning
